src/components/heading/demo/weight.twig

```diff
@@ -0,0 +1,11 @@
+{% embed '@cloudfour/objects/rhythm/rhythm.twig' with {
+  class: 'o-rhythm--condensed'
+} only %}
+  {% block content %}
+      {% include '@cloudfour/components/heading/heading.twig' with {
+        level: 1,
+        weight: 'light',
+        content: 'c-heading--weight-' ~ 'light'
```

Comment:
Any reason we're using the `~` character to join these two separate strings instead of combining into one string?